packages/core/src/storybook/stand-alone-documentaion/shadow/card-shadow-example/card-shadow-example.tsx

```diff
@@ -11,7 +11,9 @@ export const CardShadowExample = () => {
     <Frame>
       <div className={classes[CSS_BASE_CLASS]}>
         <img src={peopleColumn} alt="" />
-        <Heading value="People Column" className={classes[`${CSS_BASE_CLASS}-title`]} type={Heading.types.h4} />
```

Comment:
We never had h4?

Reply:
We had in the old typography but we no longer have